### ISSUE TITLE
Interface Types for self defined DataTypes

### ISSUE THREAD

**Issue by realAP** [WebAssembly/interface-types#124]
Will it be possible to define a Struct of Basic Types and pass this though Interface Types?

**Comment by lukewagner** [comment]
Yes, for a classic C-style `struct` you would use the [record](https://github.com/WebAssembly/interface-types/blob/linking-rebase/proposals/interface-types/Explainer.md#interface-types) interface type.
